server/src/resources/cart/cart.repository.ts

```diff
@@ -0,0 +1,39 @@
+import { CartEntity, ICart } from './cart.interfaces';
+import { Cart } from './cart.models';
+
+class CartRepository {
+  getCarts = async (): Promise<CartEntity[]> => {
+    const carts = await Cart.find();
```

Comment:
According to task description:
contains a list of products and their amount that user wants to order 
In your case this is not included